### DISCUSSION TITLE
Clarification required - where to write test method that checks if TheoryAttribute is sealed?

### DISCUSSION THREAD

**Discussion by Joe-Dunleavy** [xunit/xunit#2554] [Question] [answered]
I was hoping to contribute to the project by implementing [#2366](https://github.com/xunit/xunit/issues/2366).

I'm unsure where I should write the test method for asserting whether or not the `TheoryAttribute` class is sealed.

The `TheoryAttribute` class was moved  from the xUnit.Extensions package (deprecated) to the the `xunit.v3.core` library, but the `Theoryattribute` tests were not migrated over to the xUnit v3 core tests.

Given that none of the existing tests in `xunit.v3.core` relate to the test method I wrote for the `TheoryAttribute` class, I added a new `ClassModifierTests` class in `xunit.v3.core.tests` and added my test method to that class. However, I'm unsure if this is the right place for it.

**Comment by bradwilson** [comment, marked as answer]
I don't think such a test is necessary. We generally don't write tests for things like this.

Thanks!
